src/zope/server/tests/test_taskthreads.py

```diff
@@ -1,6 +1,10 @@
 import doctest
 import logging
-from cStringIO import StringIO
+
+try:
+    from cStringIO import StringIO
+except ImportError:
+    from io import StringIO
```

Comment:
This seems to be used as the underlying buffer of a logging.StreamHandler. Similar comments as in test_wsgiserver.py (when it's used to replace sys.stdXXX) apply.